### DISCUSSION TITLE
Take a photo from the Photos app, not the local camera

### DISCUSSION THREAD

**Discussion by crKRoEYS** [ente-io/ente#4544] [Enhancements]
To ensure that photos are protected from access, e.g. by the operating system and other apps, as soon as they are created, it would be very good if you could also take photos from the Photos app. In this case, they should be stored directly in the Ente Cloud and never locally on the smartphone. Just as an example: This function already exists in other cloud apps, such as Tresorit.

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/967
